core/src/main/scala/com/broadinstitute/dsp/CheckRunner.scala

```diff
@@ -38,6 +39,7 @@ trait CheckRunner[F[_], A] {
         .parEvalMapUnordered(50)(rt => checkResource(rt, isDryRun).handleErrorWith(_ => F.pure(None)))
         .unNone
         .map(_.toString)
+        .evalTap(_ => dependencies.metrics.incrementCounter(s"${appName}/${configs.checkType}"))
```

Comment:
`evalTap` is similar to `observe`, it performs some effects but doesn't change the original Stream